en/08_Changelogs/6.2.0.md

```diff
@@ -52,6 +52,20 @@ If your own code uses a `font-icon-*` CSS class, we recommend adding `aria-hidde
 
 - [`Hierarchy::duplicateWithChildren()`](api:SilverStripe\ORM\Hierarchy\Hierarchy::duplicateWithChildren()) used to set new sort values for duplicated children - it now uses the sort values from the original records.
 - Fields returned from [`QueuedJobDescriptor::getCMSFields()`](api:Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor::getCMSFields()) have changed to improve the UX. If you were changing those fields (e.g. with a subclass or an extension implementing `updateCMSFields()`) you may need to adjust your code to account for these changes.
+- The text for various buttons in the CMS have changed from "Add {record}" to "Add new {record}". The text uses new localisation keys:
+
+    |old key|new key|
+    |---|---|
+    |`SilverStripe\Forms\GridField\GridField.Add`|`SilverStripe\Forms\GridField\GridField.AddNew`|
+    |`AssetAdmin.ADD_FOLDER_BUTTON`|`AssetAdmin.ADD_NEW_FOLDER_BUTTON`|
+    |`CampaignAdmin.ADDCAMPAIGN`|`CampaignAdmin.ADDNEWCAMPAIGN`|
+    |`SilverStripe\UserForms.ADDEMAILRECIPIENT`|`SilverStripe\UserForms.ADDNEWEMAILRECIPIENT`|
```

Comment:
```suggestion
```
Doesn't exist? - `SilverStripe\UserForms\Model\UserDefinedForm` which you have below is correct

Reply:
Both exist:
- https://github.com/silverstripe/silverstripe-userforms/blob/8af55ff2b4dc33f75ce58c07bf98489b7e835677/lang/en.yml#L4-L6
- https://github.com/silverstripe/silverstripe-userforms/blob/8af55ff2b4dc33f75ce58c07bf98489b7e835677/lang/en.yml#L358-L360